llvm/test/CodeGen/AArch64/sme-framelower-use-bp.ll

```diff
@@ -65,7 +65,8 @@ define void @quux() #1 {
 ; CHECK-NEXT:    mov sp, x9
 ; CHECK-NEXT:    sub x10, x29, #104
 ; CHECK-NEXT:    stur x9, [x10, #-256] // 8-byte Folded Spill
-; CHECK-NEXT:    addvl x9, x8, #1
+; CHECK-NEXT:    mov x9, x8
+; CHECK-NEXT:    incb x9
```

Comment:
This can still increase code size as exemplified here.
I'm not sure how I can avoid it without doing the fold later.

Reply:
I think that's a good idea, thanks. I'll give that a try and update the PR.

Reply:
I've added `HasFastIncVL` (and `FeatureDisableFastIncVL`) to control this - is this what you had in mind?
I'm happy to invert the logic of `FeatureDisableFastIncVL` if you'd rather have it enabled on a CPU-by-CPU basis (presumably, based on the SWOGs, at least for the Neoverse V2 and Neoverse V3).